tests/test_main.py

```diff
@@ -47,6 +47,11 @@ def test_global_arrival(self):
         self.assertTrue(lambda: callable(settings.PRISTINE_FUNCTION_SETTING))
         self.assertEqual(settings.PRE_SETUP_TEST_SETTING, 6)
 
+    def test_imagekit_app_config_working(self):
+        """Test whether app-config is correctly injecting the imagekit settings."""
+        from django.conf import settings
```

Comment:
You could test on the `INSTALLED_APP`, to ensure the test is correctly set up, with:
```python
        self.assertIn('imagekit', settings.INSTALLED_APPS)
```